admin/README.md

```diff
@@ -40,3 +41,9 @@ To add a new endpoint:
 2. Re-generate gRPC and OpenAPI interfaces by running `make proto.generate`
 3. Copy the new handler signature from the `AdminServiceServer` interface in `proto/gen/rill/admin/v1/api_grpc_pb.go`
 4. Paste the handler signature and implement it in a relevant file in `admin/server/`
+
+### CLI login
+For trying out CLI login add auth-url parameter to point to local admin server like this:
+```
+go run ./cli auth login --auth-url http://localhost:8080/
```

Comment:
why do we need a `token` flag for this ? I see it in the code

Reply:
It's just a way to override the default – usually see it in other CLIs, could be useful for testing or when we have service tokens in the future

Reply:
might be needed for logout, if we delete the token on admin server

Reply:
> It's just a way to override the default (i.e. loading from ~/.rill) – usually see it in other CLIs, could be useful for testing or when we have service tokens in the future

Yes but probably should be added only for commands where it can be used. For login and logout I don't see the use, it will be ignored anyways so it just adds to the confusion of the user. Like why do I see token flag in the login cmd help, is it needed ?

Also for commands where it will be used, `token` specified using the flag should override the one in creds file or will be used even if no token is present in creds file.  

Another question - For logout flow I was thinking we should also delete the token from the user table. So the token present in the creds file will be deleted.

Reply:
Makes sense about `logout`, you can remove it from the `auth` command group. But let's keep it on the `org` and `project` groups, it's easier to set it as a persistent flag

Reply:
For logout, you're right. We should add a gRPC endpoint for revoking the current token, e.g. `RevokeCurrentAuthToken` (`DELETE /v1/tokens/current`), that the CLI can call in `rill logout`.

Reply:
added